tensorboard/pip_package/pip_smoke_test.sh

```diff
@@ -1,238 +0,0 @@
-#
-# Note:
-#   * This script requires virtualenv.
-
-set -eu
-
-die() {
-  printf >&2 '%s\n' "$1"
-  exit 1
-}
-
-PY_VERSION=2
-TEST_PORT=0
-NUM_RETRIES=20
-while [[ "$#" -gt 0 ]]; do
-  if [[ "$1" == "--python3" ]]; then
-    PY_VERSION=3
-  elif [[ "$1" == "--retries" ]]; then
-    NUM_RETRIES="$2"
-    shift
-  elif [[ "$1" == "--port" ]]; then
-    TEST_PORT="$2"
-    shift
-  else
-    die "ERROR: Unrecognized argument $1"
-  fi
-  shift
-done
-
-echo
-echo "=== Performing smoke test of tensorboard PIP package ==="
-echo "Settings:"
-echo "  PY_VERSION=${PY_VERSION}"
-echo "  TEST_PORT=${TEST_PORT}"
-echo "  NUM_RETRIES=${NUM_RETRIES}"
-echo
-
-# Check that virtualenv is installed.
-if [[ -z "$(which virtualenv)" ]]; then
-  die "ERROR: virtualenv is required, but does not appear to be installed."
-fi
-
-PIP_TMP_DIR=$(mktemp -d --suffix _tensorboard)
-
-echo
-echo "Building tensorboard pip package in directory: ${PIP_TMP_DIR}"
-echo
-
-cd "$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
-bazel build tensorboard/pip_package:build_pip_package
-
-# Create virtualenv directory, cleanly (i.e., no --system-site-packages).
-VENV_TMP_DIR=$(mktemp -d --suffix _tensorboard_venv)
-
-echo
-echo "Creating virtualenv directory at: ${VENV_TMP_DIR}"
-echo
-
-if [[ "${PY_VERSION}" == 2 ]]; then
-  virtualenv -p python "${VENV_TMP_DIR}"
-elif [[ "${PY_VERSION}" == 3 ]]; then
-  virtualenv -p python3 "${VENV_TMP_DIR}"
-fi
-
-echo
-echo "Activating virtualenv at ${VENV_TMP_DIR}"
-echo
-
-export VIRTUAL_ENV="${VENV_TMP_DIR}"
-export VENV_BIN_DIR="${VENV_TMP_DIR}/bin"
-export PATH="${VENV_BIN_DIR}:${PATH}"
-unset PYTHON_HOME
-
-echo
-echo "Installing and upgrading pip packages required for wheel building"
-echo
-
-pip install --upgrade pip setuptools wheel
-
-echo
-echo "Creating tensorboard pip package in directory: ${PIP_TMP_DIR}"
-echo
-
-bazel-bin/tensorboard/pip_package/build_pip_package "${PIP_TMP_DIR}"
-
-# Install the dependency, tensorflow, first.
-echo
-echo "Installing nightly tensorflow pip package."
-echo
-
-pip install tf-nightly
-
-echo
-echo "Installing the just-built tensorboard pip package"
-echo
-
-if [[ "${PY_VERSION}" == 2 ]]; then
-  pip install "${PIP_TMP_DIR}"/tensorboard*-py2-*.whl
-elif [[ "${PY_VERSION}" == 3 ]]; then
-  pip install "${PIP_TMP_DIR}"/tensorboard*-py3-*.whl
-fi
-
-echo
-echo "Calling tensorboard public python APIs"
-echo
-
-# Check that we can now use TensorBoard's public python APIs as installed with
-# the pip package. To do this we must cd away from the bazel workspace directory
-# to one that doesn't have a local 'tensorboard' python module hierarchy.
-TEST_API_CALL="
-import tensorboard as tb
-tb.summary.scalar_pb('test', 42)
-from tensorboard.plugins.projector import visualize_embeddings
-from tensorboard.plugins.beholder import Beholder, BeholderHook
-"
-
-echo "  python>>> $TEST_API_CALL"
-echo
-(cd "${VENV_TMP_DIR}" && python -c "$TEST_API_CALL")
-
-# Check tensorboard binary path.
```

Comment:
I think this is also something it'd be good to preserve in the new smoke test.  This ensures that the tensorboard binary is actually coming from the freshly built pip package.  Otherwise, if we fail to build that binary correctly, then the smoke test might still pass by falling back to a system-wide installed tensorboard binary.